src/CC-BY-2.5-AU.xml

```diff
@@ -0,0 +1,403 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="CC-BY-2.5-AU"
+            name="Creative Commons Attribution 2.5 Australia">
```

Comment:
```suggestion
            name="Creative Commons Attribution 2.5 Australia" listVersionAdded="3.14">
```

Reply:
This attribute is a relatively recent addition to the XML schema for the License List, so some of the older license XML submissions don't contain it. That being said, it's recommended for new submissions.